be/src/olap/memtable_flush_executor.cpp

```diff
@@ -132,10 +132,12 @@ void MemTableFlushExecutor::init(const std::vector<DataDir*>& data_dirs) {
 }
 
 // NOTE: we use SERIAL mode here to ensure all mem-tables from one tablet are flushed in order.
+// unique key merge on write should flush serial cause calc delete bitmap should load segment serial
```

Comment:
This comment should move to `be/src/olap/delta_writer.cpp:L143`

Reply:
done